docs/index.asciidoc

```diff
@@ -181,6 +181,14 @@ URI to proxy server if required
 The name of the target SQS queue. Note that this is just the name of the
 queue, not the URL or ARN.
 
+[id="plugins-{type}s-{plugin}-queue_owner_aws_account_id"]
+===== `queue_owner_aws_account_id`
+
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+The owning account id of the target SQS queue.
+
```

Comment:
Maybe worth adding a note that the queue owner must grant permissions to access this queue.

Reply:
Yes, this was a good call because I discovered both accounts need to grant permissions for this to work and that's not immediately obvious if you aren't familiar with cross account IAM permissions.